content/docs/command-reference/exp/list.md

```diff
@@ -10,7 +10,7 @@ usage: dvc exp list [-h] [-q | -v] [--rev <rev>]
                     [git_remote]
 
 positional arguments:
-  git_remote    Optional Git remote name or repo URL
+  git_remote    Optional Git remote name or authenticated Git URL
```

Comment:
Why not just `Git URL`? Using `authenticated` here (and in the other places this change was added) doesn't make sense to me. This also makes it seem confusing since we don't mention authentication at all for `Git remote name`, but we do for `Git URL` (even though the potential auth requirements apply in both situations).

In the docs for `remote add`/`remote modify` we document URLs as

> `url` - remote location

We don't say

> `url` - authenticated remote location